### PR TITLE
nimdsphax

### PR DESCRIPTION
**Pull request by lifehackerhansol**
**Description**
- Added nimdsphax.
- Renamed homebrew-launcher-* to installing-boot9strap-* along with integration of nimdsphax.
- Dropped USM / Fredtool / Frogtool, Bannerbomb3 now uses menuhax -> nimdsphax/uloader depending on system version.
- General system stability improvements were made to enhance the user's experience.

### CODE REVIEW

**Review by lilyuwuu — APPROVED** [review]
everything lgtm!

File: _pages/en_US/bannerbomb3-(legacy).txt [hunk 1]
```diff
@@ -24,12 +24,17 @@ These instructions work on USA, Europe, Japan, and Korea region consoles as indi
...
 {% include_relative include/bb3-prep.txt %}
 
+#### Section II - BannerBomb3
+
+{% include_relative include/menuhax67-prep-bb3.txt %}
+
 ___
 
-### Continue to [Installing boot9strap (menuhax)](installing-boot9strap-(menuhax))
+### Continue to [Installing boot9strap (menuhax (Legacy))](installing-boot9strap-(menuhax-(legacy)))
```
[the remark this answers is not in the document]
Comment (lifehackerhansol) [reply _pages/en_US/bannerbomb3-(legacy).txt:39]: `menuhax, Legacy`, then? like with bb3

[the remark this answers is not in the document]
Comment (lilyuwuu) [reply _pages/en_US/bannerbomb3-(legacy).txt:39]: yeah sure that would be better

[the remark this answers is not in the document]
Comment (lifehackerhansol) [reply _pages/en_US/bannerbomb3-(legacy).txt]: Amended

**Pull request merged**